static/js/apps/eval_retrieval_generation/feedback_navigation.tsx

```diff
@@ -156,8 +156,8 @@ export function FeedbackNavigation(
       while (!(targetId in userQuery)) {
         targetId -= 1;
       }
-      setSessionQueryId(targetId);
       setFeedbackStage(getFirstFeedbackStage(evalType));
+      setSessionQueryId(targetId);
```

Comment:
why the order here matters?

Reply:
I think it has to do with the pane listening on both feedback stage and session query id changes and I'm not sure it'll be better, but just switched it to the same order as eval list choosing a new query

Reply:
i think react setState is applied async and in batch, so the order here should not matter.